Description: Fix ntpdig
 ntpdig (and by extension, the ntpdate wrapper) does not correctly set the
 time.
Forwarded: https://gitlab.com/NTPsec/ntpsec/issues/407
Applied-Upstream: https://gitlab.com/NTPsec/ntpsec/commit/4d8d801c3829ca2dad287c502c3aef3cb5a736ef
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2017-11-06
--- a/ntpclients/ntpdig
+++ b/ntpclients/ntpdig
@@ -427,7 +427,8 @@
                       % (hexstamp(pkt.t1()), hexstamp(pkt.t2())))
                 print("xmt t3: %s dst t4: %s"
                       % (hexstamp(pkt.t3()), hexstamp(pkt.t4())))
-                pkt.posixize()
+            pkt.posixize()
+            if debug:
                 print("org t1: %f rec t2: %f" % (pkt.t1(), pkt.t2()))
                 print("xmt t3: %f dst t4: %f" % (pkt.t3(), pkt.t4()))
                 print("rec-org t21: %f  xmt-dst t34: %f"
